source/_posts/2018-01-08-how-to-quickly-deploy-a-vuejs-app-to-heroku.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "How to quickly deploy a VueJS app to Heroku"
+category: javascript
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle: jmbejar
+  image: /images/team/jorge-bejar.jpg
+  description: Love to explore and write about Ruby, Javascript, full-stack development and technology in general.
+---
+
+Recently, I was investing some time learning VueJS and I found that it is a very interesting framework to play with. In fact, I've been working on a new project prototype during a few days and wanted to show it to some people, so I wanted to publish it anywhere in the Internet.
+
+I decided to put the project on Heroku so I started to research what is the best way to do that. To my surprise, I didn't find much about it apart from a few posts like [this](https://codeburst.io/quick-n-clean-way-to-deploy-vue-webpack-apps-on-heroku-b522d3904bc8) and [this](https://medium.com/netscape/deploying-a-vue-js-2-x-app-to-heroku-in-5-steps-tutorial-a69845ace489). 
+
+<!-- more -->
+
+Assuming that a Heroku account is already created and the VueJS project already exists, the approach explained in those articles could be sumarized in the following steps:
+
+* Write a minimal NodeJS web server using Express
+* Build the assets locally
+* Add the `dist` folder to the Git repository, so it is included when pushing to Heroku
+
+What I didn’t like of this solution was the need to build locally the site and check in the changes within the `dist` folder. I wanted to have this step handled by Heroku when pushing a new version of my application.
```

Comment:
`didn't like about`?

Reply:
to build locally the site -> to build the site locally?

Reply:
`check-in`